sqlite_scanner.cpp

```diff
@@ -242,6 +242,37 @@ static unique_ptr<GlobalTableFunctionState> SqliteInitGlobalState(ClientContext
 	return move(result);
 }
 
+static string SqliteTypeToString(int sqlite_type) {
+	switch (sqlite_type) {
+	case SQLITE_ANY:
+		return "any";
+	case SQLITE_INTEGER:
+		return "integer";
+	case SQLITE_TEXT:
+		return "text";
+	case SQLITE_BLOB:
+		return "blob";
+	case SQLITE_FLOAT:
+		return "float";
+	default:
+		return "unknown";
+	}
+}
+
+static void AssertTypeMatches(int sqlite_column_type, int expected_type) {
+	if (sqlite_column_type != expected_type) {
```

Comment:
Can we include (1) the name of the column, and (2) the value that we found, in the error message?